odoo/addons/base/models/res_users.py

```diff
@@ -537,6 +537,9 @@ def _search(self, args, offset=0, limit=None, order=None, count=False, access_ri
 
     @api.model_create_multi
     def create(self, vals_list):
+        for values in vals_list:
```

Comment:
I did not see anything in upgrade PR about this so I'm guessing we don't care about existing users?

Side note: commit message has a typo: "shoudl".